redaxo/src/core/lib/console/package/activate.php

```diff
@@ -24,16 +24,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $package = rex_package::get($packageId);
         if ($package instanceof rex_null_package) {
             $io->error('Package "'.$packageId.'" doesn\'t exists!');
-            return;
+            return 1;
         }
 
         $manager = rex_package_manager::factory($package);
         $success = $manager->activate();
         $message = strip_tags($manager->getMessage());
```

Comment:
in welcher sprache kommt diese message?
haben wir in der cli per default en_US eingestellt @gharlan ?

oder sehen wir hier auf englisch beschriebene commandos dann deutsche/anderssprachige responses?

Reply:
Habe das gerade noch einmal getestet, die kommen in der ausgewählten Backendsprache.